### ISSUE TITLE
bug(android): font cropped when long pressing the key on the landscape view.

### ISSUE THREAD

**Issue by dinakaranr** [keymanapp/keyman#11849]
### Describe the bug

The letters are partially hidden in the popup and sub-menu.

### Reproduce the bug

Environments:
Android: 14 simulators and physical devices
Keyman version: 17.0.326
Keyboard: EuroLatin (SIL) 3.0.2

Steps to reproduce: 
Installed the "Keyman 17.0.326" version and gave all permissions to the application.
Checked the "Enable Keyman as system-wide keyboard" and set the keyboard as the default keyboard box on the settings page.
Open the Keyman app. Enable the "Predictions" and install the "Dictionary."
Enter some sentences in the text area.
Rotate the mobile portrait to a landscape view.
Press any of the "y, g, p, j, t" letter keys and hold for a few seconds.
Actual Results: Here, a letter cropped on the popup after pressing and holding the key.
Please refer to the screenshot.
<img width="300" src="https://github.com/keymanapp/keyman/assets/19683059/c70e4069-9401-4f84-b9cc-500d16fdb7c5">
Note: This behavior appeared in physical device too.

### Keyman apps

- [X] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.326

### Device

Andorid 14 physical device

### Keyboard name

EuroLatin (SIL)

**Comment by darcywong00** [comment]
I think this is a trade-off of using the keyboard in landscape orientation (which is not a good UX).
We used to have a taller default keyboard size in orientation, but it covered most of the screen, making it hard to type.
On the other hand, now that we have a shorter keyboard to see more screen, all the keys are squished.

As a workaround, the user can always go into the "Adjust keyboard height" menu and make the landscape keyboard taller (so the longpress keys aren't squished)...

**Comment by mcdurdin** [comment]
> I think this is a trade-off of using the keyboard in landscape orientation (which is not a good UX).

Not really. The font should be scaling to fit the bounding box. At extremes, this may be unreadably small (which is not a good UX), but in the screenshots given, a properly scaled font should be readable.
